packages/compass-sidebar/src/components-legacy/sidebar/navigation-items.jsx

```diff
@@ -31,7 +31,7 @@ const navItemButton = css({
   backgroundColor: uiColors.gray.dark2,
   ':hover': {
     cursor: 'pointer',
-    backgroundColor: uiColors.gray.dark3,
+    fontWeight: 'bold',
```

Comment:
is this ok? seems to be for the old sidebar

Reply:
I'll add a comment, but I added that in because the old sidebar is already affected by the compass-databases-navigation changes and I'm trying to not break it too much in the meantime.